Source/Request/Request.JSONP.js

```diff
@@ -119,19 +119,20 @@ Request.JSONP = new Class({
 		return !!this.running;
 	},
 
-	clear: function(){
+	clear: function(index){
```

Comment:
Hmmm.

What if the index isn't passed in? If it's required, that's a breaking change.

What if the index passed in isn't the running request?